README.md

```diff
@@ -1,74 +1,169 @@
-# Hello Microverse
 
-**Hello Microverse** project is a project based on implementing new features on future projects.
 
-# Built With
-  Client:
-  * HTML.
-  * CSS.
-  
-  Server:
-  * none
+# 📖 Hello Microverse 
 
-Key Features
 
- **Text styling**
 
-# Live Demo
-[Link to Live DEMO](https://calebchris000.github.io/Hello-World/)
+**Hello Microverse** is a project created to add new features.
+
+## 🛠 Built With HTML and CSS
+
+### Tech Stack
+
+
+
+ **HTML Markup**
+
+   [HTML](https://www.w3.org/)
+
+
+
+  **Styling**
+
+  [CSS](https://web.dev/learn/css/)
+
+
+
+# Features
+
+### Text styling
+
+
+- **Style Text**
+- **Colorize Text**
+- **Size Text**
+
+
+
+
+## 🚀 Live Demo
+
+
+- [Live Demo Link](https://calebchris000.github.io/Hello-World/)
+
+
+
+## 💻 Getting Started
+
+
 
-# Getting Started:
 To get a local copy up and running, follow these steps.
 
-# Prerequisites
+### Prerequisites
 
 In order to run this project you need:
-1. **Git**.
-2. **A text editor. (Visual Studio Code, Atom, Sublime Text.)**
-3. **NodeJS.**
 
 
-# Setup
-* Install git by running `sudo apt install git`
-* Install Visual Studio Code by running `sudo apt install visual-studio-code`
-* Install nodejs by running `sudo apt install npm`
-* Clone this repository to your desired folder: `git clone https://github.com/calebchris000/Hello-World.git`
+```sh
+ sudo apt install npm
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+### Install
+
+Install this project with: 
+
+
+
+```sh
+  cd my-project
+  npm install
+```
+
 
-# Install
-Install this project with: linter
+### Usage
 
-Usage
-To run the project, execute the following command: `npm run index.html`
+To run the project, execute the following command:
 
 
-# Run tests
+```sh
+  ssh server
+```
+
+
+### Run tests
 
 To run tests, run the following command:
-Deployment
 
-You can deploy this project using:
+
+```sh
+  bin/npm test test/models/article_test.js
+```
```

Comment:
[OPTIONAL]

- KIndly remove the run test section from your readme, since you are not running test on this project 👍🏽